contracts/contracts/core/BidderRegistry.sol

```diff
@@ -179,6 +179,7 @@ contract BidderRegistry is
         uint256 amtMinusFeeAndDecay = decayedAmt - feeAmt;
 
         protocolFeeTracker.accumulatedAmount += feeAmt;
+        emit FeePayout.FundsAccumulatedForTreasury(commitmentDigest, feeAmt, protocolFeeTracker.recipient, bidState.bidder, provider);
```

Comment:
If adding this we should ensure  `protocolFeeTracker.accumulatedAmount` is emitted